package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@microsoft/vscode-serial-monitor-api",
+  "version": "0.1.0",
+  "description": "Public API for vscode-serial-monitor",
+  "keywords": [
+    "vscode",
+    "serial",
+    "api",
+    "vscode-serial-monitor",
+    "monitor",
+    "serial monitor"
+  ],
+  "bugs": {
+    "url": "https://github.com/microsoft/vscode-serial-monitor/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/vscode-serial-monitor"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Microsoft Corporation"
+  },
+  "publisher": "ms-vscode",
+  "main": "dist/api.js",
+  "typings": "dist/api.d.ts",
+  "scripts": {
+    "build": "tsc -p ./",
+    "build:dev": "tsc -p ./",
+    "clean": "rimraf ./dist",
```

Comment:
I don't think this will work without a development dependency on `rimraf`.

Reply:
Changing.